src/browser/provider/pool.js

```diff
@@ -120,6 +120,9 @@ export default {
         if (!await provider.isValidBrowserName(browserName))
             throw new GeneralError(RUNTIME_ERRORS.cannotFindBrowser, alias);
 
+        if (typeof alias === 'object' && alias.alias)
+            alias = alias.alias;
```

Comment:
In this case it seems that parsing from line 113 seems to be redundant and this `if` block could be moved to  line 112 with early return (`return alias`).

Reply:
Thank you for your review. The problem is that the '_parseAlias' method has two ways of parsing objects, and if this object doesn't have exact properties, the method should throw an error. If I rise up this condition in code and if the object contains only the 'alias' property, I'll get an exception because the object doesn't have the necessary properties.

Reply:
I meant, that this case after moving to the top could be like this:
```
       if (typeof alias === 'object' && alias.alias)
            return alias;
```

without any additional parsing, as it seems that the object that is meeting this condition is a proper, already internally prepared, browser alias object

Reply:
In the beginning, the alias contains the value that the user set in the CLI or in the config file. In the config file, you can assign the 'browsers' property as a string or an object. In this object, you can set only 'alias':
```json
{
  "browsers": {
    "alias": "chrome"
  }
}
```
And it will be a mistake.

Reply:
Indeed this is tricky and indistinguishable then.